driver/driver.go

```diff
@@ -280,6 +278,14 @@ func NewDriver(ctx context.Context, cfg *config.Driver) (Driver, error) {
 		cfg.TLS = &tls.Config{MinVersion: tls.VersionTLS12}
 	}
```

Comment:
is the min version to catch this client being used from old go runtime?

Reply:
This is to prevent connection to insecure servers running TLS1.1 and older

Reply:
reported by gosec lint.

Reply:
ok.